### PR TITLE
Translation of the concurrency.md file

### PR DESCRIPTION
**Pull request by getbrandao**
Correcting errors generated for me about pull request!

### CODE REVIEW

File: pt/lessons/advanced/concurrency.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+layout: página
```
Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: don't translate this layout, or category

Comment (getbrandao) [inline pt/lessons/advanced/concurrency.md]: okay, I will make the corrections, I'm sorry

Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: No problem o/ Here to help :beers:

**Comment by pragmaticivan** [comment]
Cool @getbrandao you need to squash the commits now. If you need help in a hangout again let me know.

**Comment by pragmaticivan** [comment]
Solid @getbrandao , Thanks :shipit: :beers:

**Comment by getbrandao** [comment]
@pragmaticivan Thank you and I'm sorry all my mistakes! :)

**Comment by doomspork** [comment]
Thank you @getbrandao :+1:

Mistakes are welcome, it's how we learn :grinning:

**Pull request merged**